Configurations/Sanitizers.xcconfig

```diff
@@ -82,3 +82,7 @@ WK_XCODE_VERSION_AFTER_13_3_1700 = YES;
 // -fno-delete-null-pointer-checks: do not let the compiler remove nullptr checks that could otherwise be removed because they are considered undefined behavior.
 // -fno-optimize-sibling-calls: disable tail call elimination for more accurate crash stacks.
 WK_UNDEFINED_BEHAVIOR_SANITIZER_OTHER_CFLAGS_YES = -fno-delete-null-pointer-checks -fno-optimize-sibling-calls -fno-sanitize=vptr -fsanitize=enum,return;
+
+// Libfuzzer
+
+// Nothing extra is necessary for libfuzzer!
```

Comment:
This isn't right. We need:

```
WK_LIBFUZZER_OTHER_CFLAGS_YES = -fsanitize-coverage=inline-8bit-counters,trace-cmp -fno-sanitize-coverage=pc-table -Wno-error;
```

(and maybe something similar for ldflags too?)